### ISSUE TITLE
SDP for SIPCallDescriptor

### ISSUE THREAD

**Issue by acscorrea** [sipsorcery/sipsorcery#214]
I'm trying to connect to one endpoint with 
`SIPClientUserAgent.Call(SIPCallDescriptor)`

The code works fine if i set the SDP option to `null` in the SIPCallDescriptor, but when i try to pass my custom SDP, it changes to `Via: SIP/2.0/UDP 0.0.0.0`.

The `SDP.ToString` looks correct, it's equal to another one generated with another lib that works fine. Have tried to handwrite the SDP in a string, same error.

**Comment by sipsorcery** [comment]
Are you basing your code on the [UserAgentClient](https://github.com/sipsorcery/sipsorcery/tree/master/examples/UserAgentClient) example or one of the `Getting Started` examples?

If it's the former and you're attempting to use the `SIPUserAgent` class then you've encountered a breaking change that I need to fix. If you want to set a custom SDP you'll need to use the `SIPClientUserAgent` class (it's one half of the `SIPUserAgent` class under the hood). But if you do that then you'll have to manually take care of coordinating the SDP you get back from the remote party with the `RTPSession`. If you just want to test the signalling out and don't care about the media handling then it's fine but if you're expecting the audio and video processing to auto-magically get managed based on a custom SDP then it won't work.

**Comment by acscorrea** [comment]
I'm coding based on the `UserAgentClient`. 
I did not try the `Getting Started` code because i need both Windows and Linux.

Right now I'm using `SIPClientUserAgent`, attempting to do signalling. I will handle the packages and the media, but in the moment i can't even send the SDP.

Running the same code, without any changes, I got it working one time. Can't say it's a network problem, neither can describe what leads to fail or successs.  I will continue the tests for more information
